packages/v4/src/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -3,5 +3,92 @@ id: Tree view
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **tree view** is a structure that displays data in a hierarchical view. It can be used in a [dropdown](/components/dropdown), [drawer](/components/drawer), [primary-detail](/demos/primary-detail), [modal](/components/modal), or [wizard](/components/wizard).
+
+A default tree view provides a tree structure with items and an arrow to expand or collapse child items. Every tree view can be extended with these [optional elements](#tree-view-capabilities).
+
+## Tree view elements
+
+A default tree view consists of 5 elements:
+
+<img src="./img/tree-view-elements.png" width="332" />
+
+1. **Expand/collapse button:** A button that allows you to expand or collapse child nodes 
+2. **Parent node:** A node that contains other items (for example, a folder that contains multiple documents)
+3. **Child node:** A node within the parent node (for example, a document within a folder)
+4. **Leaf node:** The last item in a tree, which doesn't have any child nodes
```

Comment:
Does this have to be the last item in a tree? Isn't this just any items that didn't start from inside a folder?

Reply:
Yeah, it can be confusing. What about this? **The leaf node is a node with no children.**

Reply:
Just a quick note that we'd want to make the description a fragment (not a full sentence) so that it's consistent with the other descriptions in this list. So could we do "Leaf node: A node without children" ?

Reply:
YES! :) Works for me